### ISSUE TITLE
`getTranslations` is supported OOTB in i18n Ally

### ISSUE THREAD

**Issue by amannn** [ixartz/Next-js-Boilerplate#338]
Hey @ixartz,

just wanted to give you a heads-up that https://github.com/lokalise/i18n-ally/pull/1048 was finally released on the VSCode marketplace. It contains the fix for `getTranslations` to work without having to manually configure anything:

https://github.com/ixartz/Next-js-Boilerplate/blob/642d3f395bf8602529411778d228cb45364c9917/.vscode/i18n-ally-custom-framework.yml#L7-L10

Maybe you'd like to give it a go to potentially clean up your boilerplate!

**Comment by ixartz** [comment]
Hey @amannn,

Thank you so much for the heads-up. Last time I have tried, the PR was merged but I was surprised it was released/published.

This is a really great news.

I'll clean it today.

**Comment by amannn** [comment]
Yep, it unfortunately took some time due to publishing issues, but it's finally out now! 😄

Thanks again for suggesting the regex that also matches `namespace:`, the merged PR in i18n Ally uses your proposal!

**Comment by ixartz** [comment]
I just open the PR and merge it: https://github.com/ixartz/Next-js-Boilerplate/pull/339/files

Now, it's much more cleaner.

Extremely happy to see my suggestion being used in the release.

Thank you so much for reporting it @amannn and thank you for `next-intl`
